administrator/components/com_admin/sql/updates/postgresql/6.0.0-2025-10-11.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "#__history"
+    ADD COLUMN "is_current" SMALLINT NOT NULL DEFAULT 0;
```

Comment:
```suggestion
    ADD COLUMN "is_current" SMALLINT NOT NULL DEFAULT 0 /** CAN FAIL **/;
```
Please add `/** CAN FAIL **/` installer hint so it does not cause a broken update in case if the column already exists for some reason.